src/batchPdf.test.js

```diff
@@ -0,0 +1,33 @@
+const certUrl = "./test/fixtures/undigestedCerts/example.0.json";
+const pdfUrl = "./test/fixtures/pdf";
```

Comment:
```suggestion
const PDF_FOLDER_PATH = "./test/fixtures/pdf";
```

`path`  is clearer compared to `url`